_documentation/user/flags/jumping.md

```diff
@@ -5,4 +5,17 @@ type: Good
 summary: Tank can jump. Use Tab key. Can't steer in the air.
 ---
 
-...
+The Jumping (JP) flag allows the player to jump from the ground or any object that is touching the bottom side of the tank. An exception are objects with the attributes `drivethrough` or `passable`, where the result is the tank passing through the object.
+
+A lot of maps utilize the default jumping option to already allow the effect of Jumping without the tanks being required to hold the specific flag for it. The Jumping flag holds meaning and effect on maps that do not utilize said option.
+
+## Tactics
+
+Jumping is potent on maps filled with objects, as it allows for more spatial maneuvering. It can be used for either pursuit or escaping purposes, as a player can open up new paths by jumping on or over objects.
+
+On pursuit, it can assist in taking aerial shortcuts to reach the target destinations faster, or ensure a better battle position. While escaping, it can help rabbits move out of hunters' line of fire in the rabbit hunt game mode; it can assist team flag runners to progress further towards their own base while being shot at in the capture the flag game mode; or it can allow tanks to run from opponents holding dangerous close proximity flags, such as [Shock Wave](../shock-wave/).
+
+## Server Settings
+
+- `jumpVelocity` - the speed a tank moves vertically while jumping
```

Comment:
```suggestion
- `_jumpVelocity` - the speed a tank moves vertically while jumping
```

Reply:
@allejo we have to agree on a standardized format here. Wings didn't include `_` in their `server settings` section, so either they should or Jumping shouldn't.

Reply:
Gah! Oversight on my part, the `_` should belong in the Wings docs too.

Reply:
Added `_`